packages/app/src/systems/Bridge/components/BridgeSteps/BridgeSteps.tsx

```diff
@@ -0,0 +1,75 @@
+import { cssObj } from '@fuel-ui/css';
+import { Flex, Icon, Stack, Box, Text } from '@fuel-ui/react';
+
+type Step = {
+  name: string;
+  status: string;
```

Comment:
it's useful to accept `ReactNode` instead of `string` for this 2..

in case we need to insert icons or other different elements